diagnostic.md

```diff
@@ -5,74 +5,97 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend is used to store data, send data, and process requests sent from a client or front-end server. It is used to help do the processing of applications and allow for "persistent state" storage.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The controller uses the Model layer to fetch data
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The Controller communicates with the Model.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+We don't use view since our browser or client acts as the view layer.  It interprets the data
+sent and renders it in the browser.
```

Comment:
More importantly we manage view states on the front end because it allows us to lessen the requests we make to the server.